Model/Sales/MagentoOrderCollection.php

```diff
@@ -240,12 +239,16 @@ public function setFulfilment(): self
                 $order->setPickupLocation($pickupLocation);
             }
 
+            $weight = 0;
+
             foreach ($this->order->getItems() as $magentoOrderItem) {
                 $orderLine = new OrderLineOptionsFromOrderAdapter($magentoOrderItem);
+                $weight    += $orderLine->getProduct()->getWeight();
 
                 $orderLines->push($orderLine);
             }
 
+            $order->setWeight($weight);
```

Comment:
Deze functie bestaat bij mij niet, mogelijk is dat een wijziging in de sdk ergens (is dat zo?), maar in de customsdeclaration wordt ook een hele totalweight berekend op basis van producten. Waarom is dit dan op deze manier gedaan nog extra erbovenop?